Core/src/main/java/de/tor/tribes/io/ServerManager.java

```diff
@@ -115,6 +116,7 @@ public static void loadServerList(Proxy pProxy) throws Exception {
 
             bytes = isr.read(data);
             sum += bytes;
+            //TODO why this??
```

Comment:
No idea, probably added due to timing issues in the past, but seems to make no sense.